CMakeLists.txt

```diff
@@ -16,12 +16,40 @@ ExternalProject_Add(googletest
   LOG_BUILD         ON
 )
 
+include("cmake/define_option.cmake")
+
+define_option(ROC_INCLUDE_DIR "" STRING "roc toolkit include directory")
+define_option(ROC_LIB_DIR "" STRING "roc toolkit library directory")
+define_option(DOWNLOAD_ROC ON BOOL "automatically download and build roc-toolkit")
+
+if(DOWNLOAD_ROC)
+  include("cmake/download_roc.cmake")
+else()
```

Comment:
I suggest to move ROC_TOOLKIT_BRANCH here:

```cmake
if(DOWNLOAD_ROC)
  define_option(ROC_TOOLKIT_BRANCH "master" STRING "roc-toolkit branch")
  include("cmake/download_roc.cmake")
else()
```

this way all options will be defined in one place.

Also, since other options use just "ROC", not "ROC_TOOLKIT", I think it should be "ROC_BRANCH".

Reply:
done